domains/hareesh.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Hareesh108",
+        "email": "hareeshbhittam@gmail.com"
+    },
+    "record": {
+        "URL": "https://me-hazel-delta.vercel.app"
```

If you tryin to connect your Vercel website, this record should be `CNAME` and point to your Vercel subdomain thing, please read [the docs](https://docs.is-a.dev/).
```suggestion
        "CNAME": "me-hazel-delta.vercel.app"
```

---

@JustDeveloper1 Hii 

pls have a look, i have updated PR.